proto/projects/project.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package lavanet.lava.projects;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/lavanet/lava/x/projects/types";
+
+message Project {
+    string index = 1; // unique id that will be the combination of subscription address and project name, cannot be changed once created
+    string subscription = 2; // the subscription address that owns the project
+    string description = 3; // the description of the project for the users convinient
+    bool enabled = 4; // enabled flag
+    repeated ProjectKey project_keys = 5 [(gogoproto.nullable) = false];; // list of the projects keys
+    Policy policy = 6 [(gogoproto.nullable) = false];;
+    uint64 used_cu = 7;
+}
+
+message ProjectKey {
```

Comment:
why not call this ProjectUser with two types of users: ADMIN and DEVELOPER. It's more intuitive

Reply:
we might have more types in the future, this is to manage it the best way, these are the keys that will need to sign the txs to interact with the project